src/App.tsx

```diff
@@ -67,18 +53,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && loading && !err && posts.length === 0 && (
```

Comment:
The condition `loading && !err && posts.length === 0` might not be the best way to check for 'No posts yet'. It assumes that loading is still true, which might not be the case when checking for posts. Consider checking this condition after loading is complete, i.e., `!loading && !err && posts.length === 0`.